x-pack/platform/plugins/shared/fleet/server/tasks/sync_integrations/compare_synced_integrations.ts

```diff
@@ -216,11 +217,28 @@ const compareIntegrations = (
           latestFailedAttemptTime = `at ${new Date(
             latestInstallFailedAttempts.created_at
           ).toUTCString()}`;
-          latestFailedAttempt = latestInstallFailedAttempts.error?.message
-            ? `error: ${latestInstallFailedAttempts.error?.message}`
-            : '';
-        }
 
+          // handling special case for those integrations that cannot be found in registry
+          if (latestInstallFailedAttempts.error?.name === 'PackageNotFoundError') {
+            return {
+              ...baseIntegrationData,
+              install_status: {
+                main: ccrIntegration.install_status,
+                remote: 'not_installed',
+              },
+              updated_at: ccrIntegration.updated_at,
+              sync_status: SyncStatus.WARNING,
+              warning: {
+                title: `${ccrIntegration.package_name} can't be automatically synced`,
```

Comment:
Can we show the package title here instead of package name?

Reply:
I changed slightly the message, it wasn't easy to get the title in there without doing an additional `getPackageInfo` call:

<img width="861" alt="Screenshot 2025-06-11 at 16 26 47" src="https://github.com/user-attachments/assets/11134f75-edb9-439c-a549-f6b9816f9a1d" />